src/issuance.rs

```diff
@@ -543,7 +536,7 @@ pub enum Error {
 
     /// Verification errors:
     /// Invalid signature.
-    IssueBundleInvalidSignature(reddsa::Error),
+    IssueBundleInvalidSignature(schnorr::Error),
```

Comment:
I switched to the `schnorr:Error`, but at present I don't ever use the `source` field inside it. This might benefit from improvement.

Reply:
In addition, the schnorr::Error is opaque to avoid side channel attack ( see [k256 doc](https://docs.rs/k256/latest/k256/schnorr/struct.Error.html))

Reply:
If we decide to keep `schnorr::Error` in `IssueBundleInvalidSignature`, we would simplify the `eq` function
```
impl PartialEq for Error {
    fn eq(&self, other: &Self) -> bool {
        match (self, other) {
            (
                IssueBundleInvalidSignature(schnorr::Error::InvalidSignature),
                IssueBundleInvalidSignature(schnorr::Error::InvalidSignature),
            ) => true,
            (IssueBundleInvalidSignature(_), IssueBundleInvalidSignature(_)) => false,
            (err1, err2) => err1.eq(err2),
        }
    }
}
```

Reply:
Removed the internal `schnorr::Error` field.